drivers/bluetooth/hci/userchan.c

```diff
@@ -289,9 +332,24 @@ static void add_btuserchan_arg(void)
 
 static void btuserchan_check_arg(void)
 {
-	if (bt_dev_index < 0) {
-		posix_print_error_and_exit("Error: Bluetooth device missing. "
-					   "Specify one using --bt-dev=hciN\n");
+	if (!arg_found) {
+		posix_print_error_and_exit("Error: Bluetooth device missing.\n"
+									"Specify either a local hci interface --bt-dev=hciN\n"
```

Comment:
This indentation is quite off. note tabs are 8 chars with Zephyr's coding style.
(also below in the other 2 errors)